doc/releases/release-notes-1.14.rst

```diff
@@ -13,6 +13,39 @@ Major enhancements with this release include:
 
 The following sections provide detailed lists of changes by component.
 
+Security Vulnerability Related
+******************************
+
+The following security vulnerabilities (CVEs) were addressed in this release:
+
+* Tinycrypt HMAC-PRNG implementation doesn't take the HMAC state
+  clearing into account as it performs the HMAC operations, thereby using a
+  incorrect HMAC key for some of the HMAC operations.
+  CVE-2017-14200
+
+* The shell DNS command can cause unpredictable results due to misuse of stack
+  variables.
+  CVE-2017-14201
+
+* The shell implementation does not protect against buffer overruns resulting
+  in unpredictable behavior.
+  CVE-2017-14202
+
+* We introduced Kernel Page Table Isolation, a technique for
```

Comment:
This is the fix for Rogue Data Cache Load (CVE-2017-5754), can mention the CVE here.

Reply:
I added the CVE and forced pushed the update.